install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -1926,8 +1926,7 @@ prometheus:
     trustCRDsExist: false
 
   # -- Metrics that should be enabled or disabled from the default metric list.
-  # The list is expected to be separated by a space. (+metric_foo to enable
-  # metric_foo , -metric_bar to disable metric_bar).
+  # (+metric_foo to enable metric_foo , -metric_bar to disable metric_bar).
```

Comment:
It should be noted here that the value is an array, with the example matching that format.

```
    metrics:
      - +metric_foo
      - -metric_bar
  
This enables `metric_foo` and disables `metric_bar`.
```

Reply:
Updated. Changed the type of the property `metrics` from `string` to `array` by setting the default from `~` to `[]`.

This shouldn't change any behaviour. It's mainly that it's displayed properly in the docs too.